submissions/Roka20012/memory-pair-game/js/script.js

```diff
@@ -0,0 +1,112 @@
+const delayForAnswer = 1000;
+const delay = 500;
+const twoCards = 2;
+const winGame = 12;
+
+let flipper = document.querySelector(".flipper");
+let allCards = document.querySelectorAll(".flipper > div");
+
+function showCards() {
+    setTimeout(function() {
+        allCards.forEach(function(el) {
+            el.classList.toggle("check");
+        });
+    }, delay);
+
+    setTimeout(function() {
+        allCards.forEach(function(el) {
+            el.classList.toggle("check");
+        });
+    }, delay * 4);
+}
+
+function addCards() {
+    let img = document.querySelectorAll(".flipper div img");
+    let cards = [];
+    cards = cards.concat(shuffleCards(), shuffleCards());
+
+    cards.forEach(function(el, i) {
+        img[i].src = el;
+    });
+}
+
+function shuffleCards() {
+    let cards = ["images/man1.jpg", "images/man2.jpg", "images/man3.jpg",
+        "images/man4.jpg", "images/man5.jpg", "images/man6.jpg"
+    ];
+
+    return cards.sort(function() {
+        return 0.5 - Math.random();
+    });
+}
+
+function checkGameWin() {
+    let hideCards = document.querySelectorAll(".hide");
+
+    if (hideCards.length === winGame) {
+        let restart = document.createElement("div");
+
+        restart.innerHTML = "!!!YOU WIN!!!\n<div class='win'>RESTART</div>";
+        restart.classList.add("win-game");
+        document.body.appendChild(restart);
+
+        document.querySelector(".win").onclick = function() {
+            restart.remove();
```

Comment:
It's unnecessary but it's better to use `eventListener` instead of  `.onClick`

Reply:
Already done)